pandas/core/generic.py

```diff
@@ -4035,17 +4077,20 @@ def _reindex_multi(self, axes, copy, fill_value):
 
             .. versionadded:: 0.21.0 (list-like tolerance)
 
-        Examples
-        --------
-        >>> df.reindex_axis(['A', 'B', 'C'], axis=1)
-
         See Also
         --------
-        reindex, reindex_like
+        DataFrame.set_index : set row labels
+        DataFrame.reset_index : remove row labels or move them to new columns
+        DataFrame.reindex : change to new indices or expand indices
+        DataFrame.reindex_like : change to same indices as other DataFrame
 
         Returns
         -------
         reindexed : %(klass)s
+
+        Examples
+        --------
+        >>> df.reindex_axis(['A', 'B', 'C'], axis=1)
```

Comment:
Can you turn this into an actual example?

Reply:
It seems like 'reindex_axis' will be deprecated in the future so I did not bother with it.
```
FutureWarning: '.reindex_axis' is deprecated and will be removed in a future version. Use '.reindex' instead.
```